### ISSUE TITLE
Feature idea: ability to edit saved boards

### ISSUE THREAD

**Issue by bryanlandia** [nasser/boards#9]
Ramsey, this is such a cool tool!  Thank you.  I've been using it for note taking when I want to cut and paste from my screen.  I'd love to be able to continue work on a board though.  Would want to be able to drag a saved .html Board file back onto the Electron app and pick up where I left off.  

:)

**Comment by nasser** [comment]
Yeah! This is not terribly hard to implement, either. I will give it a shot.
